tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "removeComments": true,
     "skipLibCheck": true,
     "sourceMap": true,
-    "target": "ES5",
+    "target": "es2015",

```

Comment:
Erhm, this might cause a lot of problem, should probably not touch this.

Reply:
Yeah, changing the target would break the project in browsers that do not support es6/es2015.

Instead you could specify es2015 in the `lib` option if you want to use `async/await` or promise. By default, when the lib is not specified, if the target is es5, then the lib will be es5. However, you can set it to something else. Further info:

https://www.typescriptlang.org/docs/handbook/compiler-options.html